categories/reporting-solutions/index.md

```diff
@@ -0,0 +1,12 @@
+---
+type: top-category
+title: Reporting Solutions
+uri: reporting-solutions
+index:
+- rules-to-better-access-databases
+- rules-to-better-reporting-services
+- rules-to-better-power-bi
+- rules-to-better-business-intelligence
+- powerbi-template-apps
```

Comment:
Hi @isaaclombardssw 

This rule should be added to a category, instead of top-category

- [ ] 1. Please fix

Reply:
Done :) FYI that's Isaac's handle